src/VisualStudio/Core/Def/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -500,7 +494,8 @@ protected override bool TryExec(Guid commandGroup, uint commandId)
                                 // asynchronously added to the FindReferences window as they are computed.  The user
                                 // also knows something is happening as the window, with the progress-banner will pop up
                                 // immediately.
-                                _ = FindReferencesAsync(_streamingPresenter, symbolListItem, project);
+                                var streamingPresenter = ComponentModel.GetService<IStreamingFindUsagesPresenter>();
+                                _ = FindReferencesAsync(streamingPresenter, symbolListItem, project);
```

Comment:
❗ The life asynchronous operations must be captured at the call site, either by awaiting or by using a listener. We haven't figured out a way to analyze this cleanly, so it has to be seen in a review.